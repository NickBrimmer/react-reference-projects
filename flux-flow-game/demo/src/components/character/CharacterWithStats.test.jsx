import React from 'react';
import { shallow } from 'enzyme';
import CharacterWithStats from './CharacterWithStats';

describe('CharacterWithStats component', () => {
  it('renders CharacterWithStats', () => {
    const character = {
      image: 'http://image.png',
      health: 100,
      attack: 5,
      experience: 0
    };
    const wrapper = shallow(<CharacterWithStats character={character} />);
    expect(wrapper).toMatchSnapshot();
  });
});
