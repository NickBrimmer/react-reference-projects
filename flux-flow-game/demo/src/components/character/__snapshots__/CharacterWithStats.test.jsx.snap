// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterWithStats component renders CharacterWithStats 1`] = `
<section>
  <Character
    image="http://image.png"
  />
  <Stats
    attack={5}
    experience={0}
    health={100}
  />
</section>
`;
